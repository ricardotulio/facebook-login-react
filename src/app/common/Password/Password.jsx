import React from 'react'

import Input from '../Input'

const Password = (props) => {
  const typePassword = {
    type: "password",
    ...props,
  }

  return (
    <Input {...typePassword} />
  )
}

export default Password