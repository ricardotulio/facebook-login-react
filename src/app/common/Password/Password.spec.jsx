import React from 'react'
import renderer from 'react-test-renderer'

import Password from './Password'

test('Properties are passed element', () => {
  const password = renderer.create(
    <Password id="password-id" name="password-name" />
  )
  let tree = password.toJSON()

  expect(tree).toMatchSnapshot()
})

test('Type property cannot be overwritten', () => {
  const password = renderer.create(
    <Password id="password-id" name="password-name" type="text" />
  )
  let tree = password.toJSON()

  expect(tree).toMatchSnapshot()
})