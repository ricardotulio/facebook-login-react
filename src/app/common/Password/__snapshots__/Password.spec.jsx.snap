// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Properties are passed element 1`] = `
<input
  id="password-id"
  name="password-name"
  type="password"
/>
`;

exports[`Type property cannot be overwritten 1`] = `
<input
  id="password-id"
  name="password-name"
  type="text"
/>
`;
